packages/context-entries/lib/Keys.ts

```diff
@@ -208,6 +208,13 @@ export const KeysQueryOperation = {
   localizeBlankNodes: new ActionContextKey<boolean>('@comunica/actor-query-operation:localizeBlankNodes'),
 };
 
+export const KeysRdfParse = {
+  /**
+   * @range {boolean}
+   */
+  parseNonRecommendedFormats: new ActionContextKey<boolean>('@comunica/actor-rdf-parse:parseNonRecommendedFormats'),
```

Comment:
Instead of a new context option, should we consider exposing this parser in an alternative config of the query engine?
Since the vast majority of users will not enable this option, it might be unnecessary for them to have this parser dependency installed.